lib/maily/email.rb

```diff
@@ -105,5 +109,24 @@ def update_template(new_content, part = nil)
         f.write(new_content)
       end
     end
+
+    def versions
+      regexp = Regexp.new("^#{self.name}:")
+      self.mailer.emails.select do |email_key, _email|
+        email_key.match(regexp)
+      end
+    end
+
+    class << self
```

Comment:
as a god practice, class methods should be listed first in the class: https://github.com/rubocop/ruby-style-guide#consistent-classes